UnitsNet/CustomCode/UnitsNetSetup.cs

```diff
@@ -88,8 +83,5 @@ public UnitsNetSetup(IEnumerable<QuantityInfo> quantityInfos, UnitConverter unit
     /// <summary>
     ///     The quantities and units that are loaded.
     /// </summary>
-    /// <remarks>
-    ///     Access type is <c>internal</c> until this class is matured and ready for external use.
-    /// </remarks>
-    internal QuantityInfoLookup QuantityInfoLookup { get; }
```

Comment:
I haven't got any burning need to rename this, but it would perhaps make it more consistent if we used `Quantities` as the property name.